recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ requirements:
   build:
     - {{ stdlib("c") }}
     - {{ compiler("fortran") }}
-    - cmake
+    - cmake >=3.12
```

Comment:
You're already getting the newest CMake anyway, however, newer policies (in CMake terminology) will still not take effect, unless the CMake minimal version as defined in `CMakeLists.txt` is raised. This would need a patch of the sources (or a new upstream release with that change).